src/System.Private.CoreLib/shared/System/Number.BigInteger.cs

```diff
@@ -198,6 +406,217 @@ public static int Compare(ref BigInteger lhs, ref BigInteger rhs)
                 return 0;
             }
 
+            public static uint CountSignificantBits(uint value)
+            {
+                return (value != 0) ? (1 + LogBase2(value)) : 0;
```

Comment:
There are a few places, like this, where HWIntrinsics could probably be used in a later PR.